ui/src/components/Notifications.tsx

```
@@ -0,0 +1,50 @@
+import { Toast, Alert, Button } from 'react-daisyui';
+import { DynamicToastChild } from './Tabs';
+import { Dispatch, SetStateAction } from 'react';
+
+const daisyUiAlertClass = (status: string) => {
+  switch (status) {
+    case 'info':
+      return 'daisyui-alert-info';
+    case 'success':
+      return 'daisyui-alert-success';
+    case 'warning':
+      return 'daisyui-alert-warning';
+    case 'error':
+      return 'daisyui-alert-error';
+    default:
+      return '';
```

this looks like a silent failure. is that by design?